src/node-lib/curriculum-api-2023/queries/curriculumHeader/curriculumHeader.query.ts

```diff
@@ -0,0 +1,41 @@
+import OakError from "../../../../errors/OakError";
+import { Sdk } from "../../sdk";
+
+const curriculumHeaderQuery =
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  (sdk: Sdk) => async (args: { slug: string }) => {
+    const splitSlug = args.slug.split("-");
+    if (splitSlug.length < 2) {
+      throw new OakError({ code: "curriculum-api/not-found" });
+    }
+    const [subjectSlug, phaseSlug, examBoardSlug] = splitSlug;
+
+    if (!subjectSlug || !phaseSlug) {
+      throw new OakError({ code: "curriculum-api/not-found" });
+    }
+    const capitalise = (word: string) =>
+      word.charAt(0).toUpperCase() + word.slice(1);
+    const subject = {
+      title: capitalise(subjectSlug),
+      slug: subjectSlug,
+    };
+    const phase = {
+      title: capitalise(phaseSlug),
+      slug: phaseSlug,
+    };
+    const examBoard = {
+      title: examBoardSlug ? capitalise(examBoardSlug) : "",
```

Comment:
@sonalisin This doesn't work for exam boards that are acronyms (e.g. AQA becomes `Aqa` in the breadcrumb):
![Uploading Screenshot 2023-08-18 at 09.59.03.png…]()

Reply:
Yep - this was just placeholder - data and proper exam board info will be coming from the backend!